SPECS/kernel/config

```diff
@@ -1854,10 +1849,10 @@ CONFIG_MAC80211_LEDS=y
 # CONFIG_MAC80211_DEBUG_MENU is not set
 CONFIG_MAC80211_STA_HASH_MAX_SIZE=0
 # CONFIG_RFKILL is not set
-CONFIG_NET_9P=m
-CONFIG_NET_9P_FD=m
-CONFIG_NET_9P_VIRTIO=m
-CONFIG_NET_9P_XEN=m
+CONFIG_NET_9P=y
+CONFIG_NET_9P_FD=y
+CONFIG_NET_9P_VIRTIO=y
```

Comment:
We are simultaneously trying to figure out the set of kernel modules to include in our initramfs. Dracut is rather pedantic about this; I'm pretty sure it'll complain if a module stops existing (even if it is built-in). 

https://github.com/microsoft/azurelinux/pull/9144/files#diff-dddc34dca2f7bebb34fd880ce85f99a512ca0e73d5e2ecf571b0b741ea4c07f2R2